packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -232,6 +232,74 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
   readonly role?: iam.IRole;
 }
 
+/**
+ * The frequency at which Amazon EC2 Auto Scaling sends aggregated data to CloudWatch
+ */
+export enum Granularity {
+  /**
+   * 1Minute
+   */
+  ONE_MINUNTE = '1Minute'
+}
+
+/**
+ * Group metrics that an Amazon EC2 Auto Scaling group sends to Amazon CloudWatch.
+ */
+export enum GroupMetric {
+  /**
+   * The minimum size of the Auto Scaling group
+   */
+  MIN_SIZE = "GroupMinSize",
+  /**
+   * The maximum size of the Auto Scaling group
+   */
+  MAX_SIZE = "GroupMaxSize",
+  /**
+   * The number of instances that the Auto Scaling group attempts to maintain
+   */
+  DESIERED_CAPACITY =  "GroupDesiredCapacity",
+  /**
+   * The number of instances that are running as part of the Auto Scaling group
+   * This metric does not include instances that are pending or terminating
+   */
+  IN_SERVICE_INSTANCES = "GroupInServiceInstances",
+  /**
+   * The number of instances that are pending
+   * A pending instance is not yet in serviceThis metric does not include instances that are in service or terminating
+   */
+  PENDING_INSTANCES = "GroupPendingInstances",
+  /**
+   * The number of instances that are in a Standby state
+   * Instances in this state are still running but are not actively in service
+   */
+  STANDBY_INSTANCES = "GroupStandbyInstances",
+  /**
+   * The number of instances that are in the process of terminating
+   * This metric does not include instances that are in service or pending
+   */
+  TERMINATING_INSTANCES = "GroupTerminatingInstances",
+  /**
+   * The total number of instances in the Auto Scaling group
+   * This metric identifies the number of instances that are in service, pending, and terminating
+   */
+  TOTAL_INSTANCES = "GroupTotalInstances",
+}
+
+export interface MetricsCollectionOptions {
+  /**
+   * The frequency at which Amazon EC2 Auto Scaling sends aggregated data to CloudWatch.
+   * @default 1Minute
+   */
+  readonly granularity?: Granularity;
+}
```

Comment:
Why model an option that has only one possible value, i.e., one minute, if that's the default and the only option?